SPECS/sudo/sudo-ldap-netgroup-query.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/plugins/sudoers/ldap_conf.c b/plugins/sudoers/ldap_conf.c
```

Comment:
Let's use the commit that supersedes the original change - https://github.com/sudo-project/sudo/commit/5fbf7a3625cc199e8f350af88c1120e45fa17bf4
Will help with this change